Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -384,6 +383,30 @@ public static CommandLineArgumentsResult ProcessCommandLineArguments(string[] ar
       return isVerified;
     }
 
+    private static void WriteTrailer(PipelineStatistics stats) {
+      if (CommandLineOptions.Clo.vcVariety != CommandLineOptions.VCVariety.Doomed && !CommandLineOptions.Clo.Verify && stats.ErrorCount == 0) {
```

Comment:
Note that there are two flags, the Boogie flag `/noVerify` and the Dafny flag `/dafnyVerify:<n>`. Should this `if` test inspect both?

Reply:
No. If /dafnyVerify:0 is set, control never gets to the point of this informational message.